app/main.py

```diff
@@ -189,5 +189,6 @@ def __init__(self, *args, **kwargs):
 init_session()
 middleware = [JSONTranslator(), DatabaseSessionManager(db_session), CORSMiddleware()]
 application = App(middleware=middleware)
+application.req_options.strip_url_path_trailing_slash = True
```

Comment:
Default Settings is False.
  OK: /test
  NG: /test/
Set to True to allow NG pattern.